hack/jenkins/release_build_and_upload.sh

```diff
@@ -90,8 +90,14 @@ make checksum
 # unversioned names to avoid updating upstream Kubernetes documentation each release
 cp "out/minikube_${DEB_VERSION}-0_amd64.deb" out/minikube_latest_amd64.deb
 cp "out/minikube_${DEB_VERSION}-0_arm64.deb" out/minikube_latest_arm64.deb
+cp "out/minikube-${DEB_VERSION}-0_armhf.deb" out/minikube_latest_armhf.deb
+cp "out/minikube-${DEB_VERSION}-0_ppc64el.deb" out/minikube_latest_ppc64el.deb
+cp "out/minikube-${DEB_VERSION}-0_s390x.deb" out/minikube_latest_s390x.deb
 cp "out/minikube-${RPM_VERSION}-0.x86_64.rpm" out/minikube-latest.x86_64.rpm
 cp "out/minikube-${RPM_VERSION}-0.aarch64.rpm" out/minikube-latest.aarch64.rpm
+cp "out/minikube-${RPM_VERSION}-0.armv7hl.rpm" out/minikube-latest.armv7hl.rpm
+cp "out/minikube-${RPM_VERSION}-0.ppc64le.rpm" out/minikube-latest.ppc64le.rpm
+cp "out/minikube-${RPM_VERSION}-0.s390x.rpm" out/minikube-latest.s390x.rpm
```

Comment:
Note that we don't currently cp the deb & rpm packages for the other arches to the`latest` copy.

Meaning links like https://storage.googleapis.com/minikube/releases/latest/minikube_latest_armhf.deb will not work.

So I added these in the release script. The docs will only provide binary download instructions for now. Once they are available I can add those installation instructions in docs.

Reply:
how about we do this Part in a separate PR ?

Reply:
It's perfectly fine to **only** document packages for amd64 and arm64

You could redirect the others to the "latest" github page or something
https://github.com/kubernetes/minikube/releases/latest

"For other architectures, see **here** for additional minikube binaries"

Reply:
It's quite possible that we will only do binaries for those architectures.

Reply:
Thanks! I reverted the changes in 120925eb7ba1080eb1320b8524219e3e96892767, and added a link to the release page:

![Screen Shot 2021-05-18 at 1 52 18 AM](https://user-images.githubusercontent.com/1311594/118598180-af02f880-b77b-11eb-9ee5-6c76ffe3e092.png)